combat-trainer.lic

```diff
@@ -1154,6 +1158,12 @@ class AbilityProcess
   end
 
   private
+  def setup_barb_buff_flags
+    @barb_buffs.each do |name|
+      Flags.add("ap-#{name}-active", "#{get_data('spells').barb_abilities[name]['expired_message']}")
```

Comment:
I'd also like to see `ability_data = get_data('spells').barb_abilities[name]` extracted here